Git-commit: 3b776460d0f5f086cb9560a110ecf9b318a5f4d8
From: Xiangyou Xie <xiexiangyou@huawei.com>
Date: Fri, 19 Feb 2021 15:14:35 +0800
Subject: [PATCH] config: enable CONFIG_CPU_IDLE_GOV_HALTPOLL and
 CONFIG_HALTPOLL_CPUIDLE for arm
Patch-mainline: Queued in openEuler repo, version 5.10.0-4.1.0
References: openEuler-21.03
Git-repo: https://gitee.com/openeuler/kernel.git

hulk inclusion
category: config
bugzilla: 47727
CVE: NA

------------------------------

We enable haltpoll by default for the improvement of performance.
X86 has been supported. Now, we will provide it on ARM.

Signed-off-by: Xiangyou Xie <xiexiangyou@huawei.com>
Signed-off-by: Peng Liang <liangpeng10@huawei.com>
Reviewed-by: Hanjun Guo <guohanjun@huawei.com>
Signed-off-by: Zheng Zengkai <zhengzengkai@huawei.com>
Signed-off-by: Kai Liu <kai.liu@suse.com>
---
 arch/arm64/configs/openeuler_defconfig | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/arch/arm64/configs/openeuler_defconfig b/arch/arm64/configs/openeuler_defconfig
index c0e3d1510167..c290ea0aaa85 100644
--- a/arch/arm64/configs/openeuler_defconfig
+++ b/arch/arm64/configs/openeuler_defconfig
@@ -526,12 +526,14 @@ CONFIG_CPU_IDLE_MULTIPLE_DRIVERS=y
 # CONFIG_CPU_IDLE_GOV_LADDER is not set
 CONFIG_CPU_IDLE_GOV_MENU=y
 CONFIG_CPU_IDLE_GOV_TEO=y
+CONFIG_CPU_IDLE_GOV_HALTPOLL=y
 CONFIG_DT_IDLE_STATES=y
 
 #
 # ARM CPU Idle Drivers
 #
 CONFIG_ARM_CPUIDLE=y
+CONFIG_HALTPOLL_CPUIDLE=y
 # CONFIG_ARM_PSCI_CPUIDLE is not set
 # end of ARM CPU Idle Drivers
 # end of CPU Idle
-- 
2.31.1

